bpf/bpf_lxc.c

```diff
@@ -195,8 +195,21 @@ static __always_inline int handle_ipv6_from_lxc(struct __ctx_buff *ctx, __u32 *d
 		/* The map value is zeroed so the map update didn't happen somehow. */
 		return DROP_INVALID_TC_BUFFER;
 
+	/* Before upstream commit d71962f3e627 (4.18), map helpers were not
+	 * allowed to access map values directly. So for those older kernels,
+	 * we need to copy the data to the stack first.
+	 * We don't have a probe for that, but the bpf_fib_lookup helper was
+	 * introduced in the same release.
+	 */
+#if HAVE_PROG_TYPE_HELPER(sched_cls, bpf_fib_lookup)
```

Comment:
Can we hide this behind some `HAVE_DIRECT_ACCESS_TO_MAP_VALUES` (that expands to `HAVE_PROG_TYPE_HELPER(...)`), so that we don't need this comment everywhere?